docs/limitations.md

```diff
@@ -277,7 +277,7 @@ enabling use of `--net=host` when necessary.
 It should be noted, currently passing the `--net=host` option into a
 Clear Container may result in the Clear Container networking setup
 modifying, re-configuring and therefore possibly breaking the host
-networking setup. Do not use `--host=net` with Clear Containers.
+networking setup. Do not use `--net=host` with Clear Containers.
```

Comment:
Good catch @guangxuli! :smile:

In fact, I wonder if we should really be specifying the full option name too (`--network=`). What do you think @mcastelino  and @klynnrif ?

Reply:
Actually - let's just land this fix as it's consistent with what is already written. If we decide to only use the full option name, that can be dealt with on a separate PR.